shared/fs/index.js

```diff
@@ -21,6 +21,11 @@ type FolderProps = {
   items: Array<Types.Path>,
   path: Types.Path,
   progress: 'pending' | 'loaded',
+  sortSetting: Types._SortSetting,
+  setSortByNameAsc: () => void,
+  setSortByNameDesc: () => void,
+  setSortByTimeAsc: () => void,
+  setSortByTimeDesc: () => void,
```

Comment:
I thought all this is going away...

Reply:
Good catch; Thanks! I forgot to remove them. Only `path` is needed here.